tools/pd-api-bench/cases/cases.go

```diff
@@ -366,3 +378,86 @@ func generateKeyForSimulator(id int, keyLen int) []byte {
 	copy(k, fmt.Sprintf("%010d", id))
 	return k
 }
+
+type getKV struct {
+	*baseCase
+}
+
+func newGetKV() func() ETCDCase {
+	return func() ETCDCase {
+		return &getKV{
+			baseCase: &baseCase{
+				name: "Get",
+				cfg:  newConfig(),
+			},
+		}
+	}
+}
+
+func (c *getKV) Unary(ctx context.Context, cli *clientv3.Client) error {
+	_, err := cli.Get(ctx, "/test/0001", clientv3.WithPrefix())
```

Comment:
I think the key can be cluster id or others that must exist in etcd.

Reply:
It's okay. But I also have an idea, which is to initialize some key values

Reply:
sounds good to me